spec/interfaceSpec.js

```diff
@@ -0,0 +1,8 @@
+window.onload = function() {
+  describe('Interface', function () {
+    it('gets information from the api request and displays them individually', function () {
+      expect(document.getElementById("index").contentDocument.getElementsByClassName("expand").length)
+      .toEqual(list.getList().length)
+    })
+  })
+}
```

Comment:
Does this simply check the number of articles? 

Some thoughts on other tests: 
- Could test if the expand function works 
- Could test if the API call is made to the guardian, i.e. test your fetch function (this could be a nice integration of a mock)